README.md

```diff
@@ -24,6 +24,14 @@ A 500 response body will be returned if any issues occur when calling the Lambda
 * numberofrefs parameter isn't an int
 * any dynamodb Exceptions (key not found, unable to update, etc)
 
+The number of references that can be returned in a single call is limited, because:
+* prevent a single call using up all possible reference;
```

Comment:
A minor rewording here, change from `all possible reference` to `all possible references`